iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -40,6 +40,7 @@ public enum ErrorCode
         JobNotFound = 404002,
         PartitionNotFound = 404003,
         ModuleNotFound = 404010,
+        DeviceNotOnline = 404103,
```

Comment:
How is this value defined? Is it documented anywhere?

Reply:
this code is returned by IoT Hub when device is not online